fw/cache.c

```diff
@@ -376,7 +376,7 @@ tfw_release_node_cpus(void)
 	if(!c_nodes)
 		return;
 
-	for(node = 0; node < nr_online_nodes; node++) {
+	for_each_node_with_cpus(node) {
```

Comment:
It seems that during kernel loading we can also use `for_each_node_with_cpus`

Reply:
During kernel loading we can't use `for_each_node_with_cpus`, we do memory reserve too early, at this stage we can't get `nr_cpus` by node, it always be zero. It must be implemented in other way.